docs: Enclave pool explanation (#927)
kurtosis-tech/kurtosis: docs/docs/guides/running-in-k8s.md

:::tip Kurtosis Kloud Early Access
To switch back to using Kurtosis locally, simply use: `kurtosis cluster set docker`
:::

> V. Activate the enclave pool to accelerate the enclave creation time

Reviewer: ```suggestion
V. [Optional] Activate the enclave pool to accelerate the enclave creation time
```

--------------------------------

> This step is optional, but we recommend to do it in order to improve the user experience during the enclave creation.

Reviewer: ```suggestion
This step is optional, but we recommend taking it as it improves the user experience during the enclave creation, specifically regarding speed.
```

> Creating a new enclave from scratch demands several time-consuming engine's tasks and resources creation that can't be omitted.

Reviewer: ```suggestion
Creating a new enclave from scratch demands several time-consuming engine tasks and the creation of resources.
```

> Kurtosis includes the enclave pool feature that users can enable for reducing the enclave creation time by creating enclaves before these are requested.

Reviewer: ```suggestion
The enclave pool feature was introduced to reduce the time it takes for a user to run a Kurtosis package in the cloud by spinning up the enclaves before they are needed.
```

> The enclave pool is a functionality of the Kurtosis engine that automatically creates `idle` enclaves, since the engine is started, that will be available to be used when users need to create a new enclave (e.g: when running the `kurtosis enclave add` command)

Reviewer: ```suggestion
The enclave pool is a functionality of the Kurtosis engine that automatically creates `idle` enclaves, when the engine is started, that are then used whenever users need to create a new enclave (e.g: when running the `kurtosis enclave add` command).
```

> This mechanism speed-up the enclave creation time by using a running `idle` enclave when a new enclave is requested to the engine

Reviewer: ```suggestion
This mechanism reduces enclave creation time by using a running `idle` enclave when a new enclave is requested from the engine.
```
